### ISSUE TITLE
Model is not generating accurate recommandations [QST]

### ISSUE THREAD

**Issue by Oussamakhammassi** [NVIDIA-Merlin/Transformers4Rec#770]
Hello. I've adapted t4rec model to my own e-commerce dataset and trained it but the model is not capturing the relationships between products and not giving a logical results. For the test i give him as input a transaction containing one product. The evaluation metrics also are very low(around 0.01). The loss function is decreasing but the predictions doesn't make sense. I tried many things like changing learning rate, batch size,... but doesn't work.

**Comment by lmatejka** [comment]
Hi, after adapting default r4rec settings I also receive poor recommendations (e.g. with comparison to baseline association rules). Have you explored why this happens? Im going to debug it with small prepared dataset with minimum features to se what happens.
